contributors/TEMPLATING.md

```diff
@@ -176,9 +176,23 @@ The special files and folders are:
 
 # Things worth mentioning
 
+## Recommended way to handle complex arguments in templates
+
+Most of the time you will use string arguments for templating, but sometimes you will need to add arrays, objects, bigints, etc. You can handle them however you want, but we're recommending to use the table below as a helper.
+
+| Pattern                 | Template                                                                                    | Args                                                       | Result                                                                                  |
+| ----------------------- | ------------------------------------------------------------------------------------------- | ---------------------------------------------------------- | --------------------------------------------------------------------------------------- |
+| Replace an object       | `const replacedObj = ${deepStringify(replacedObj[0])}`                                      | `const replacedObj = { key1: "Replaced", key2: "Object" }` | `const replacedObj = { key1: "Replaced", key2: "Object" }`                              |
+| Replace an array        | `const replacedArr = ${deepStringify(replacedArr[0])}`                                      | `const replacedArr = ["Replaced", "Array"]`                | `const replacedArr = ["Replaced", "Array"]`                                             |
+| Object, add new entries | `const mergedObj = ${deepStringify({ key1: "value1", key2: "value2", ...objToMerge[0] })};` | `const objToMerge = { key3: "Merged", key4: "Object" }`    | `const mergedObj = { key1: "value1", key2: "value2", key3: "Merged", key4: "Object" };` |
+| Array, add new items    | `const arrWithAdditionalItems = ${deepStringify(['a', 'b', ...arrayToSpread[0]])}`          | `const arrayToSpread = ["Spread", "This"]`                 | `const arrWithAdditionalItems = ["a", "b", "Spread", "This"]`                           |
+| BigInt, simple var      | `const bigInt = ${deepStringify(someBigInt[0])};`                                           | `const someBigInt = 123n`                                  | `const bigInt = 123n;`                                                                  |
+| Object with bigints     | `const objectWithBigInt = ${deepStringify(objectWithBigInt[0])}`                            | `const objectWithBigInt = { key1: 123n }`                  | `const objectWithBigInt = { key1: 123n };`                                              |
+|  |
```

Comment:
I think we don't need to mention this case explicitly  since it just works out of the box?

Reply:
[3f420cf](https://github.com/scaffold-eth/create-eth/pull/135/commits/3f420cf69820a0120872ba30c0efb6c65105480f)

First thought was that after people will see a line with simple bigint they can start to build something strange for objects with bigints but hopefully I'm wrong. Removed for now, thanks